wrappers/swift/Tests/SwiftWrapperTests/SwiftWrapperTests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+@testable import SwiftWrapper
+
+final class SwiftWrapperTests: XCTestCase {
+    func testExample() throws {
```

Comment:
Do we wanna keep this dummy code?

Reply:
no plan to add swift tests for now. removing it.